modules/react/popup/lib/PopupBody.tsx

```diff
@@ -8,5 +8,7 @@ export const PopupBody = createSubcomponent('div')({
   displayName: 'Popup.Body',
   modelHook: usePopupModel,
 })<ExtractProps<typeof Card.Body>>(elemProps => {
-  return <Card.Body overflowY="auto" {...elemProps} />;
+  return (
+    <Card.Body overflowY="auto" paddingX={'l'} paddingTop="s" paddingBottom="l" {...elemProps} />
```

Comment:
This is the main change, I removed the padding from `Popup.Card` and added it to the Body so that the focus ring is visible.

Reply:
Hmm, I'm seeing issues with our approach of adding vertical padding to the Body. Go to the Modal Body Overflow story and adjust the frame for the story to trigger overflow on the Body. Scroll down the Body and note how the content scrolls out of view at the top of the Body right below the Heading:

<img width="1078" alt="overflow-scrolling" src="https://user-images.githubusercontent.com/146020/188016654-abb485fe-d59a-4305-9f88-f4284eb2cf5a.png">

Reply:
We've decided a small reduction in the vertical spacing between the Heading and Body as you scroll down the Body is acceptable (up to 8px reduction given the 8px of padding in Popup.Body, see Alan's approach in the top-level PR comment).